### PR TITLE
fix: disable some unicorn rules

### PR DESCRIPTION
**Pull request by SvetlomirB**
as mentioned by @Qix- in https://github.com/parcelLab/eventbus-ingestion/pull/49 these rules don't make sense for us. 

I'm assuming that we should disable them globally, but i can also just disable them for the ts config

### CODE REVIEW

**Review by darioblanco — APPROVED** [review]
Seems ok to me, but it won't have an effect for the consumers of this library

**Comment by darioblanco** [comment]
We should disable them globally btw

File: .eslintrc.js [hunk 1]
```diff
@@ -6,5 +6,7 @@ module.exports = {
   },
   rules: {
     "unicorn/prefer-module": "off",
+    "unicorn/prefer-spread": "off",
+    "unicorn/prefer-number-properties": "off",
```
Comment (darioblanco) [inline .eslintrc.js:10]: And this ignores the linter in this repo, you should ignore these in base.js

**Review by darioblanco — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (SvetlomirB) [reply .eslintrc.js:10]: done

Comment (darioblanco) [inline .eslintrc.js:10]: You can remove these two as they are redundant.

Comment (darioblanco) [reply .eslintrc.js:10]: I will take care of it, as the content of this PR is fine and then you can focus on other tasks

Comment (SvetlomirB) [reply .eslintrc.js]: thanks <3

File: .eslintrc.js [hunk 2]
```diff
@@ -4,7 +4,4 @@ module.exports = {
   parserOptions: {
     project: "./tsconfig.eslint.json",
   },
-  rules: {
-    "unicorn/prefer-module": "off",
```
Comment (SvetlomirB) [inline .eslintrc.js:8]: this we need, it was there before :P

Comment (darioblanco) [reply .eslintrc.js:8]: This one was already exported globally

Comment (darioblanco) [reply .eslintrc.js:8]: Why do we need to remove `prefer-module`? That rule enforces ESM (which is what we want)

Comment (SvetlomirB) [reply .eslintrc.js:8]: we were getting errors in this repo, that's why i said that, no other reason

Comment (darioblanco) [reply .eslintrc.js:8]: That is not reason enough to disable a rule, I can take a look at the errors with you and we can figure out specifically for that repo.

Therefore I will merge it as it is!

Comment (SvetlomirB) [reply .eslintrc.js:8]: Not about our repo, but this repo here

**Review by darioblanco — APPROVED** [review]

**Pull request merged**